src/client.ts

```diff
@@ -2571,6 +2610,22 @@ export class StreamChat<
     }, 500);
   }
 
+  /**
+   * encode ws url payload
+   * @private
+   * @returns json string
+   */
+  _buildWSPayload = (client_request_id?: string) => {
+    return JSON.stringify({
+      user_id: this.userID,
+      user_details: this._user,
+      user_token: this.tokenManager.getToken(),
+      server_determines_connection_id: true,
```

Comment:
this isn't used anymore

Reply:
what about user_token as part of the json payload?

Reply:
same, not needed

Reply:
authorization header is passed

Reply:
yea, will get rid of on another PR